record_imu_cam/plugin.cpp

```diff
@@ -0,0 +1,101 @@
+#include "common/plugin.hpp"
+#include "common/switchboard.hpp"
+#include "common/data_format.hpp"
+
+#include <fstream>
+#include <boost/filesystem.hpp>
+#include <iomanip>
+#include <opencv2/core/mat.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/imgproc.hpp>
+
+using namespace ILLIXR;
+/*
+The plugin to record the IMU and the camera data. It is written by Ying Jing and Henry Che from Illinois.
+*/
```

Comment:
Please add a proper description of what the plugin does. As for authorship, it is tracked in `CONTRIBUTORS.md` (and git history), so no need to add it here.